plugins/bn-avro/qml/keys/ABackspaceKey.qml

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Canonical Ltd.
```

Comment:
please do not remove this line, while adding your copyright line is recommended it is also recommended to append not replace